hr_citizenship/models/res_citizenship.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class Citizenship(models.Model):
+    _name = 'res.citizenship'
+    _order = 'sequence'
+
+    identification_id = fields.Char(
```

Comment:
does this change by nationality? Isn't this tied to the employee only?

Reply:
nationality and citizenship is different things as I start discus it here https://github.com/OCA/partner-contact/issues/709, and probably need update that mentioned addon
so if person have dual citizenship I think he would have to id for different countries 
but your point got me thinking that in one company employee should definitely have one passport in country same as company,
 and if this multi-company it can use different identification_id for reports in different companies :fire:

Reply:
in the EU, it's totally normal that people have citizenship from all over the place. In our NL based company, we have D, GR, IT and none of them has any ties with the Dutch state apart from living there

Reply:
I see, I was probably mixing up w/ the internal identification_id. Never mind :)

> but your point got me thinking that in one company employee should definitely have one passport in country same as company,
and if this multi-company it can use different identification_id for reports in different companies 

I agree w/ Holger and I'm am a good example: my passport is IT and my company is CH :wink:

Reply:
the citizenship (needs to be a one2many because too few countries allow multiple nationalities) is a property of a person, nothing to do with companies I think

Reply:
for simplification make this a computed field that is a many2one, and write in the many2many field. and add a group `Multiple nationalities` that gives a view override to show the many2many and hide the simplified field

Reply:
@hbrunn  you want separate field m2o citizenship ? I thought citizenship we keep in field  "country_id" in 'hr.employee' and 'res.citizenship'
group ok
about companies it was just thoughts, there is no restrictions now

Reply:
i'm just spitting out thoughts here. do whatever you see feasible, and it will be welcome, but reality is a hard to grasp beast, i just wanted to mention how it works in my environment